statistics/handle/handle.go

```diff
@@ -115,8 +109,6 @@ type Handle struct {
 
 	mu struct {
```

Comment:
This anonymous struct can be objectified in the subsequent implementation. At least, an anonymous struct is not a good implementation.

Reply:
Sounds good. I will take a look later.